news/10430.doc.rst

```diff
@@ -0,0 +1 @@
+Update links of setuptools
```

Comment:
Could you add more information? e.g. why do we need to change them? (because setuptools moved these documents) Also need to mention the Simple Repository link now points to PyPUG because that is the canonical place of packaging specification, and setuptools’s `easy_install` is deprecated.

Reply:
Added.